src/api-derive/staking/types.ts

```diff
@@ -88,3 +89,32 @@ type DeriveUnlocking = {
   remainingEras: BN;
   value: Balance;
 };
+
+export type TsInMs = u64;
+
```

Comment:
Do these types exist in darwinia.js? @cinlk

Reply:
The new version of darwinia.js has this type, but apps don't use this version yet

Reply:
Maybe we moved to darwinia.js as soon as possible to reduce maintenance costs